src/map.h

```diff
@@ -1235,9 +1235,10 @@ class map
         void generate( const tripoint &p, const time_point &when ) {
             generate( p.x, p.y, p.z, when );
         }
-        void place_spawns( const mongroup_id &group, int chance,
-                           int x1, int y1, int x2, int y2, float intensity,
-                           bool individual = false, bool friendly = false );
+        void place_spawns( const mongroup_id &group, const int chance,
+                           const int x1, const int y1, const int x2, const int y2, const float intensity,
+                           const bool individual = false, const bool friendly = false, const std::string name = "NONE",
+                           const int mission_id = -1 );
```

Comment:
`const` in declarations do nothing when passing by value. `const int` and `int` are indistinguishable to the compiler. It only matters in definitions and when passing by pointer/reference (`const std::string name` should become `const std::string &name` where `const` totally makes sense).

Reply:
Like how they are now? I only did it to the name and mission_id since the rest of them were like that already.

Reply:
I guess I started this right before #33230 came. Removed `const` from others except name and group.